lib/security/Authenticator.js

```diff
@@ -45,35 +49,41 @@ module.exports = oo({
 
   /*****************************************************************************
    * @method authenticate
-   * @description authenticate description
-   * @param {xxx} req -- xxx
+   * @abstract
+   * @description Authenticates the user for a request. Should be implemented by subclasses,
+   *              for example: {@link class:carbond.security.MongoDBHttpBasicAuthenticator}.
+   * @param {carbond.Request} req -- The current request
    *
-   * @returns {Object|undefined} -- This should return an object representing
+   * @returns {object|undefined} -- This should return an object representing the
    *                                user *or* undefined if the credendtials
-   *                                are missing
-   * @throws {Service.errors.Unauthorized} -- If credentials are present
-   *                                          but they fail verification.
-   * @throws {Service.errors.InternalServerError} -- If there is an exception
-   *                                                 on user lookup.
+   *                                are missing.
+   * @throws {HttpErrors.Unauthorized} -- If credentials are present
+   *                                      but they fail verification.
+   * @throws {HttpErrors.InternalServerError} -- If there is an exception
+   *                                             on user lookup.
    */
   authenticate: function(req) {
     return undefined
   },
 
   /*****************************************************************************
    * @method isRootUser
-   * @description isRootUser description
-   * @param {xxx} user -- xxx
-   * @returns {boolean} -- xxx
+   * @abstract
+   * @description Checks if a user is root.
+   * @param {object} user -- An object representing a user
+   * @returns {boolean} -- `true` if the user is determined to be root, `false` otherwise.
```

Comment:
i think double back ticks are needed to get monospaced highlighted text here (e.g., ``true``)